src/bundle/Resources/views/themes/admin/ui/error_page/404.html.twig

```diff
@@ -14,10 +14,10 @@
         <div class="error-page__image-wrapper">
             <img src="{{ asset('bundles/ezplatformadminui/img/errors/404.png') }}" />
         </div>
-        <h2 class="error-page__title">{{ 'error.page.404.title'|trans|desc('Something went wrong. Sorry!') }}</h2>
+        <h2 class="error-page__title">{{ 'error.page.404.title'|trans|desc('Page not found. Sorry!') }}</h2>
         <div class="error-page__messages">
-            <p class="error-page__message">{{ 'error.page.404.message'|trans|raw|desc('It\’s an error 404. The page you requested was not found.<br/>It might have been deleted or might be temporarily unavailable.') }}</p>
-            <p class="error-page__message">{{ 'error.page.404.actions'|trans({ '%dashboard%': path('ezplatform.dashboard'), '%search%': path('ezplatform.search') })|raw|desc('Check the URL, in case there would be an error in there.<br/>If not, go to the <a class="error-page__link" href="%dashboard%">dashboard</a> or <a class="error-page__link" href="%search%">search</a> for specific content.') }}
+            <p class="error-page__message">{{ 'error.page.404.message'|trans|raw|desc('Error 404: the page you requested was not found.<br/>It might have been deleted or is temporarily unavailable.') }}</p>
```

Comment:
```suggestion
            <p class="error-page__message">{{ 'error.page.404.message'|trans|raw|desc('Error 404: the page you requested was not found.<br/>It might have been deleted or it is temporarily unavailable.') }}</p>
```